src/IntelliTect.Coalesce/DataAnnotations/SecurityInfoProperty.cs

```diff
@@ -104,6 +119,73 @@ public bool IsReadable(ClaimsPrincipal user)
             return true;
         }
 
+        /// <summary>
```

Comment:
In general, this sort of thing (building up strings of C# code) should be done in the code generation projects, not in IntelliTect.Coalesce. There are some exceptions to this in the current codebase, but generally, those exceptions are code that predates the pure C# generators (in the past, our code generation was done with .cshtml files using razor as a templating engine - even the generation of C# files).

These annotation properties, for example, are only ever useful in the generation of the API controllers, so they should probably live pretty close to the `ModelApiController` generator, or its base class, `ApiController`. Probably as private/protected methods within one of those.

Reply:
I did this to remain consistent with the design of the `SecurityInfoClass`, this may be something I need your advice on where to move this logic @GrantErickson.